docs/document/content/user-manual/shardingsphere-jdbc/yaml-config/jdbc-driver/known-implementation/_index.en.md

```diff
@@ -213,7 +213,7 @@ Then for the intercepted fragment of the following YAML file,
 ds_1:
   dataSourceClassName: com.zaxxer.hikari.HikariDataSource
   driverClassName: $${FIXTURE_DRIVER_CLASS_NAME::com.mysql.cj.jdbc.Driver}
-  jdbcUrl: jdbc:mysql://$${FIXTURE_HOST::}:$${FIXTURE_PORT::}/$${FIXTURE_DATABASE::}?useUnicode=true&characterEncoding=UTF-8
```

Comment:
- @malonglei I noticed that this was introduced by #32817. 
- But https://dev.mysql.com/doc/connector-j/en/connector-j-reference-configuration-properties.html never mentions the existence of a `useUnicode` property. What is the source of this property?